docker/frontend-test-stack/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+services:
+  contracts:
+    build:
+      context: ../contracts
+    image: renegade-contracts:latest
+    entrypoint: sleep infinity
+  sequencer:
+    build:
+      context: ../devnet
+    volumes:
+      - deployments:/deployments
+    ports:
+      - "5050:5050"
+    environment:
+      - STARKNET_DEVNET_CAIRO_VM=rust
```

Comment:
I don't think this env var does anything for Katana, it may be an artifact from when we were using the Python-based `starknet-devnet`

Reply:
oops yes, artifact from when we were using the devnet